lib/picker.js

```diff
@@ -23,8 +23,7 @@
 var $window = $( window )
 var $document = $( document )
 var $html = $( document.documentElement )
-var supportsTransitions = document.body.style.transition != null
-
+var supportsTransitions = function() { return document.body.style.transition != null; }
```

Comment:
This change also requires a change on line 813 to call the function, rather than simply assert it is not falsy.

```js
function focusPickerOnceOpened() {

        if (IS_DEFAULT_THEME && supportsTransitions()) {
```